pkg/operation/botanist/component/etcd/etcd_test.go

```diff
@@ -89,13 +89,20 @@ var _ = Describe("Etcd", func() {
 			Begin: "1234",
 			End:   "5678",
 		}
-		now                         = time.Time{}
-		quota                       = resource.MustParse("8Gi")
-		garbageCollectionPolicy     = druidv1alpha1.GarbageCollectionPolicy(druidv1alpha1.GarbageCollectionPolicyExponential)
-		garbageCollectionPeriod     = metav1.Duration{Duration: 12 * time.Hour}
+		now                     = time.Time{}
+		quota                   = resource.MustParse("8Gi")
+		garbageCollectionPolicy = druidv1alpha1.GarbageCollectionPolicy(druidv1alpha1.GarbageCollectionPolicyExponential)
+		garbageCollectionPeriod = metav1.Duration{Duration: 12 * time.Hour}
+		compressionPolicy       = druidv1alpha1.GzipCompression
```

Comment:
```suggestion
		compressionPolicy       = druidv1alpha1.DefaultCompression
```
reason same as above.